common/src/main/java/com/redhat/cloud/notifications/openbridge/BridgeHelper.java

```diff
@@ -39,7 +43,7 @@ public class BridgeHelper {
 
     private Bridge bridgeInstance;
 
-    @ApplicationScoped
+    @RequestScoped
```

Comment:
Did you make that change because of the token? I think this should still be `@ApplicationScoped` and the token should be injected at class level instead of calling `getAuthToken()` directly in this method.